File: jungle-camping/JungleCamping.c
#include <stdio.h>
#include <string.h>
#define MAX 500

int main() {
  char *animals[] = {"Lion", "Tiger", "Snake", "Bird"};
  char *sounds[] = {"Grr", "Rawr", "Ssss", "Chirp"};
   
  char noise[MAX];
  fgets(noise, MAX, stdin);
  
  for (char *sound = strtok(noise, " "); sound; sound = strtok(NULL, " ")) 
    for (int i = 0; i < 4; ++i)
      if (strcmp(sound, sounds[i]) == 0)
   	printf("%s ", animals[i]);
   
   return 0;
}

File: jungle-camping/JungleCamping.java
import java.util.Scanner;
import java.util.Map;

public class JungleCamping
{
    public static void main(String[] args) {
        Map<String, String> animals = Map.of(
            "Grr","Lion", "Rawr", "Tiger", "Ssss", "Snake", "Chirp", "Bird");
	    
        final var input = new Scanner(System.in);
        final var noise = input.nextLine().split(" ");
        
        for (var sound : noise) {
            System.out.printf("%s ", animals.get(sound));
        }
    }
}
